.travis.yml

```diff
@@ -5,7 +5,9 @@ d:
   - ldc
 
 before_script: git fetch --unshallow
-script: "dub --config=test"
+script:
+  - "dub --config=test"
+  - cd test_package && dub build
```

Comment:
> I can't run test.d locally since Arch Linux isn't supported. Even if it were, I'd rather not download clang when I have it installed.

Sorry about that, LLVM doesn't provide pre-compiled releases for Arch Linux. I'm currently working on moving the LLVM matrix from `test.d` to the Travis CI and Appveyor build scripts. Perhaps I can add support for running the tests without downloading LLVM.